Context aware policies can be written with Rego (#289)
kubewarden/docs: docs/explanations/context-aware-policies.md

All other Kubernetes resources need the full definition: `<api>/<version>`.

:::

## Further readings

> More detailed information about context aware policies can be found inside of [this section](../writing-policies/spec/05-context-aware-policies.mdx)

Reviewer: ```suggestion
More detailed information about context aware policies can be found in [this section](../writing-policies/spec/05-context-aware-policies.mdx)
```
